### DISCUSSION TITLE
About adding LinkedIn profile link in the profile.

### DISCUSSION THREAD

**Discussion by NotShrirang** [community/community#17278] [Other Feature Feedback, Questions, & Ideas]
Hello there!
I have noticed that there is no option to add link to your LinkedIn profile.
Of course, we can add it in profile repository. (Repository with name as your username.)
But just like Twitter profile option, it would be better if LinkedIn profile option was added.

Thanks!

**Comment by ghost** [comment]
@NotShrirang Hi! Several people have requested this same feature: https://github.com/github-community/community/discussions/6601
